ext/fiddle/pointer.c

```diff
@@ -24,6 +24,7 @@ struct ptr_data {
     void *ptr;
     long size;
     freefunc_t free;
+    int freed;
```

Comment:
Is `int` and `1` and `0` the right way to do booleans in C extensions? I found some other things but wasn't sure if they were internal for MRI?

Reply:
This failed in all CI configurations when I tried it.

Reply:
We need `#include <stdbool.h>`.

Reply:
Done.